Ecal/src/Ecal/EcalClusterProducer.cxx

```diff
@@ -73,6 +72,35 @@ void EcalClusterProducer::produce(framework::Event& event) {
       cluster.addHits(wcVec[aWC].getHits());
       cluster.addFirstLayerHits(fWcVec[aWC].getHits());
 
+      float x(0), y(0), z(0), xx(0), yy(0), zz(0), n(0);
+      float w = 1;  // weight
+      float sumw = 0;
+
+      for (auto hit : wcVec[aWC].getHits()) {
+        if (hit->getEnergy() < minHitEnergy_) continue;
+        w = log(hit->getEnergy() - log(minHitEnergy_));
```

Comment:
are you really meaning to take a log of the log here? or just missed a `)`?
```suggestion
        w = log(hit->getEnergy()) - log(minHitEnergy_));
```

Reply:
thanks, updated this in DB scan builder too, where i originally found it